lib/view/pages/home_page.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:superformula_test/core/resources/extensions.dart';
+import 'package:superformula_test/view/widgets/extended_menu_fab.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({super.key});
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  double _imageScale = 0;
```

Comment:
**nit**: the state class is already private, you don't also need to make library-private variables (not a big deal though  🙂)